lib/badwords.js

```diff
@@ -35,6 +35,9 @@ class Filter {
         const wordExp = new RegExp(`\\b${word.replace(/(\W)/g, '\\$1')}\\b`, 'gi');
         return !this.exclude.includes(word.toLowerCase()) && wordExp.test(string);
       })
+      .length > 0 || this.list.filter((word) => {
```

Comment:
I think it would be a better idea to just make a new function here. Maybe `isAnyProfane`? This should also check the words against the `exclude` array.

Reply:
Okay. I'll work on this as soon as I'm able